packages/expo-image/ios/ImageView.swift

```diff
@@ -139,13 +139,10 @@ public final class ImageView: ExpoView {
     // It seems that `UIImageView` can't tint some vector graphics. If the `tintColor` prop is specified,
     // we tell the SVG coder to decode to a bitmap instead. This will become useless when we switch to SVGNative coder.
     if imageTintColor != nil {
-      context[.imageDecodeOptions] = [
-        SDImageCoderOption.webImageContext: [
-          "svgPrefersBitmap": true,
-          "svgImageSize": sdImageView.bounds.size,
-          "svgImagePreserveAspectRatio": true
-        ]
-      ]
+      var decodeOptions = context[.imageDecodeOptions] as? [SDImageCoderOption: Any] ?? [:]
+      decodeOptions[.decodeThumbnailPixelSize] = sdImageView.bounds.size
+      decodeOptions[.decodePreserveAspectRatio] = true
+      context[.imageDecodeOptions] = decodeOptions
```

Comment:
- merge `context[.imageDecodeOptions]`
- `SDImageCoderOption.webImageContext` is deprecated